pandas/tests/reductions/test_reductions.py

```diff
@@ -949,14 +949,17 @@ def test_all_any_boolean(self):
         assert s3.all(skipna=False)
         assert not s4.any(skipna=False)
 
-        # Check level TODO(GH-33449) result should also be boolean
         s = Series(
             [False, False, True, True, False, True],
             index=[0, 0, 1, 1, 2, 2],
             dtype="boolean",
         )
-        tm.assert_series_equal(s.all(level=0), Series([False, True, False]))
-        tm.assert_series_equal(s.any(level=0), Series([False, True, True]))
+        tm.assert_series_equal(
+            s.all(level=0), Series([False, True, False], dtype="boolean")
```

Comment:
Can you set result/expected on separate lines? This way it's easier to step through with a debugger.

Reply:
Yep sounds good, just ended up breaking off into a new test